docs/team/rayyan35p.md

```diff
@@ -8,3 +8,33 @@ title: Rayyan's Project Portfolio Page
 TaskHub is a desktop project management application used by project managers to manage projects and their team members in each project. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
 
 Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to prioritise projects [\#113](https://github.com/AY2324S1-CS2103T-T08-3/tp/pull/113)
+    * What it does: Allows the user to set a priority for each project.
+    * Justification: This feature allows the user to be able to more easily see which projects need to be worked on as higher priority projects will stand out.
+    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to assign and unassign tasks within projects to employees [\#156](https://github.com/AY2324S1-CS2103T-T08-3/tp/pull/156)
+    * What it does: Allows the user to assign a task in a project to an employee within that project or unassign a task that has been assigned to an employee. 
+    * Justification: This feature lets the user keep track of which employee should be doing each task.
+    * Highlights: Due to the nature of how tasks are stored in projects, and employees are stored in tasks, this was particularly complex as there were many possible cases where a command that might change an employee's details does not affect the employee within the task. All possible scenarios had to be considered to ensure there were no bugs.
+    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
```

Comment:
Are the `Credits` lines in line 15 and 20 necessary, since you did not credit anyone.